package/gluon-config-mode-contact-info/i18n/fr.po

```diff
@@ -1,33 +1,38 @@
 msgid ""
 msgstr ""
-"Content-Type: text/plain; charset=UTF-8\n"
 "Project-Id-Version: PACKAGE VERSION\n"
 "PO-Revision-Date: 2015-08-12 23:30+0100\n"
 "Last-Translator:Tobias Bernot <tqbs@airmail.cc>\n"
 "Language-Team: French\n"
 "Language: fr\n"
 "MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
 msgid "Contact info"
 msgstr "Informations de Contact"
 
 msgid ""
-"Please provide your contact information here to allow others to contact "
-"you. Note that this information will be visible <em>publicly</em> on "
-"the internet together with your node's coordinates. This means it can "
-"be downloaded and processed by anyone. This information is "
-"not required to operate a node. If you chose to enter data, it will be "
-"stored on this node and can be deleted by yourself at any time."
+"Please provide your contact information here to allow others to contact you. "
+"Note that this information will be visible <em>publicly</em> on the internet "
+"together with your node's coordinates. This means it can be downloaded and "
+"processed by anyone. This information is not required to operate a node. If "
+"you chose to enter data, it will be stored on this node and can be deleted "
+"by yourself at any time."
 msgstr ""
-"Entrez vos coordonnées ici si vous souhaitez permettre à d'autres "
-"personnes de communiquer avec vous. Ces informations seront affichées "
-"<em>en ligne</em> avec les coordonnées du nœud. Il peut être téléchargé "
-"et traité par tous. Il n'est pas obligatoire "
-"de fournir ces informations pour pouvoir opérer un nœud. Si vous avez "
-"choisi d'entrer des données, elles seront stockées sur ce nœud et vous "
-"pourrez les effacer par vous-même à tout moment."
+"Entrez vos coordonnées ici si vous souhaitez permettre à d'autres personnes "
+"de communiquer avec vous. Ces informations seront affichées <em>en ligne</"
+"em> avec les coordonnées du nœud. Il peut être téléchargé et traité par "
+"tous. Il n'est pas obligatoire de fournir ces informations pour pouvoir "
+"opérer un nœud. Si vous avez choisi d'entrer des données, elles seront "
+"stockées sur ce nœud et vous pourrez les effacer par vous-même à tout moment."
 
 msgid "e.g. E-mail or phone number"
 msgstr "Ex : E-mail ou numéro de téléphone"
+
+msgid "gluon-config-mode:contact-help"
+msgstr ""
+
+msgid "gluon-config-mode:contact-note"
+msgstr ""
```

Comment:
The `gluon-config-mode:...` (and similar) entries should be omitted. We don't have a scripted way to do that at the moment.

Reply:
Removed these msgids